pipelines/common/src/main/java/com/google/fhir/analytics/view/ViewDefinition.java

```diff
@@ -254,11 +249,17 @@ public static class Select {
     private List<Select> unionAll;
   }
 
+  @Builder(toBuilder = true)
   @Getter
   public static class Column {
     private String path;
     private String name;
     private String type;
+    private boolean collection;
+    private String description;
+    // The following fields are _not_ read from the ViewDefinition.
+    private String inferredType;
```

Comment:
Instead of storing these over here, can we infer them at runtime. I saw a comment where we are trying to move all the inference logic to one place, may be as part of that we can do this as well.

Reply:
Ok, makes sense.